web/src/components/tasks/business-formation/contacts/RegisteredAgent.tsx

```diff
@@ -266,27 +262,14 @@ export const RegisteredAgent = (): ReactElement => {
                       hasError={doesFieldHaveError("agentOfficeAddressMunicipality")}
                       type="MOBILE-ONLY"
                     >
-                      <strong>
-                        <ModifiedContent>
-                          {Config.formation.fields.agentOfficeAddressMunicipality.label}
-                        </ModifiedContent>
-                      </strong>
                       <FormationField fieldName="agentOfficeAddressMunicipality">
-                        <MunicipalityDropdown
-                          municipalities={municipalities}
+                        <BusinessFormationTextField
+                          label={Config.formation.fields.agentOfficeAddressMunicipality.label}
                           fieldName="agentOfficeAddressMunicipality"
-                          error={doesFieldHaveError("agentOfficeAddressMunicipality")}
+                          required={true}
+                          validationText={getFieldErrorLabel("agentOfficeAddressMunicipality")} // renders Office City
                           disabled={shouldBeDisabled("agentOfficeAddressMunicipality", "ADDRESS")}
```

Comment:
pebble - why did you decide to keep this prop?

Reply:
referring do the disabled

Reply:
Oversight. Thanks for catching it!

Reply:
needs to be confirmed with product.. missed A/C